sql_generators/glean_usage/common.py

```diff
@@ -18,6 +18,24 @@
 APP_LISTINGS_URL = "https://probeinfo.telemetry.mozilla.org/v2/glean/app-listings"
 PATH = Path(os.path.dirname(__file__))
 
+# added as the result of baseline checks being added to the template,
+# these apps do not receive a baseline ping causing a very basic row_count
+# check to fail. For this reason, the checks relying on this ping
+# need to be omitted. For more info see: bug-1868848
+NO_BASELINE_PING_APPS = (
+    "mozilla_vpn",
+    "mozillavpn_cirrus",
+    "accounts_backend",
+    "burnham",
+    "firefox_reality_pc",
+    "lockwise_android",
+    "mach",
+    "monitor_cirrus",
+    "moso_mastodon_backend",
+    "mozphab",
+    "mozregression",
+)
+
```

Comment:
I think we should move this into the config file here: https://github.com/mozilla/bigquery-etl/blob/6e2e8f6f688a91f292774995f119ecdd0b63cd8f/bqetl_project.yaml#L433

Reply:
@scholtzan  That sounds like a reasonable suggestion, where would be the correct place to file a ticket for this so we do not forget about it?

Reply:
There is a PR here: https://github.com/mozilla/bigquery-etl/pull/4676